plugins/Installation/FormSuperUser.php

```diff
@@ -40,9 +40,8 @@ function init()
             ->setLabel(Piwik::translate('Installation_Password'));
         $password->addRule('required', Piwik::translate('General_Required', Piwik::translate('Installation_Password')));
         $pwMinLen = UsersManager::PASSWORD_MIN_LENGTH;
-        $pwMaxLen = UsersManager::PASSWORD_MAX_LENGTH;
-        $pwLenInvalidMessage = Piwik::translate('UsersManager_ExceptionInvalidPassword', array($pwMinLen, $pwMaxLen));
-        $password->addRule('length', $pwLenInvalidMessage, array('min' => $pwMinLen, 'max' => $pwMaxLen));
+        $pwLenInvalidMessage = Piwik::translate('UsersManager_ExceptionInvalidPassword', array($pwMinLen));
```

Comment:
The translation of this key needs to be updated in `plugins/UsersManager/lang/en.json`. There you find a key `ExceptionInvalidPassword`. It could otherwise lead to warnings as parameters do not match the amount of `%s` in that string and the translation string itself doesn't match anymore.